script.js

```diff
@@ -74,8 +105,58 @@ class Library {
       }
     }
   }
+
+  myTime() {
+    const timer = document.getElementById('clock');
+    // eslint-disable-next-line no-undef
+    const datertimer = luxon.DateTime.now();
+    timer.innerText = datertimer.toFormat('MMMM dd yyyy, hh:mm:ss a');
+  }
+
+  startTimer() {
+    setInterval(() => {
+      this.myTime();
+    }, 1000);
+  }
 }
 
 let library = new Library([]);
 library.addBook();
-library.showBooks();
\ No newline at end of file
+library.showBooks();
+library.startTimer();
```

Comment:
- [ ] Your linter throws a `no-use-before-define` because this variable is declared after it's being used above. Simply move this declaration to the top of the `script.js` file, so that it is defined before it is used.